### PR TITLE
OPENEUROPA-1700: Removing drone file for the final site.

### PR DESCRIPTION
**Pull request by voidtek**
### Description

Removing drone file for the final site.

### Change log

- Added: a step to remove the drone file on the setupwizard script. 
- Added: Documentation

### CODE REVIEW

File: README.md.dist [hunk 1]
```diff
@@ -139,3 +139,10 @@ git commit -m "Initial commit."
 ```
 
 Now you are ready to push your project to your chosen code hosting service.
+
+## 4. CI on Drone
+
+To follow the continuous integration of your project, you can check it on [Drone](https://drone.fpfis.eu/).
+
+A default pipeline is created by devops (handling QA of the code, test scenarios with behat and creation ).
```
Comment (imanoleguskiza) [inline README.md.dist]: This comment is  a bit misleading. As it stands it either means you need to ask the devops to create the pipeline or that it is already created automatically: it needs clarification.

File: README.md.dist [hunk 2]
```diff
@@ -139,3 +139,10 @@ git commit -m "Initial commit."
...
+But the customization of this pipeline is possible, adding your `.drone.yml` on the root folder of the project.
```
Comment (imanoleguskiza) [inline README.md.dist]: No need to add "But", this sentence doesn't contradict what was said before. I would rephrase it like this:
"the customization of this pipeline is possible by adding a custom ".drone.yml" file to the root folder of the project.

File: README.md.dist (hunk 1 above)
Comment (voidtek) [reply README.md.dist]: :+1:

File: README.md.dist (hunk 2 above)
Comment (voidtek) [reply README.md.dist]: :+1:
